applications/CompressiblePotentialFlowApplication/tests/naca0012_small_transonic_test/naca0012_small_transonic_parameters.json

```
@@ -0,0 +1,104 @@
+{
+    "problem_data"                     : {
+        "problem_name"    : "naca0012",
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0,
+        "start_time"      : 0.0,
+        "end_time"        : 1
+    },
+    "solver_settings"                  : {
+        "solver_type"            : "potential_flow",
+        "domain_size"     : 2,
+        "model_part_name" : "MainModelPart",
+        "model_import_settings"  : {
+            "input_type"     : "mdpa",
+            "input_filename" : "naca0012_small"
```

I was changing the wrong file before, this should work in order to have just one mpda in another directory as @rubenzorrilla suggested:

```suggestion
            "input_filename" : "../naca0012_small_mdpa/naca0012_small"
```

---

Works for me as well